.travis.yml

```diff
@@ -15,6 +15,9 @@ env:
     - BQ_TEST_PROJECT_ID=luigi-travistestenvironment
     - BQ_TEST_INPUT_BUCKET=luigi-bigquery-test
     - GOOGLE_APPLICATION_CREDENTIALS=test/gcloud-credentials.json
+    - AWS_DEFAULT_REGION=us-east-1
```

Comment:
us-east-1 is defined here, but eu-west-1 is used within dynamodb_test.py...

Reply:
Yep that's just a lack of consistency from my part. I'll remove the region from dynamodb_test.py and let it resove itself from the env vars.